portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -95,6 +96,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     
     // Get the input from the form, depending on the type of commentor.
     String name = getParameter(request, "name-input", "");
+    String location = getParameter(request, "location-input", "");
     boolean student = Boolean.parseBoolean(getParameter(request, "student", "false"));
     boolean industry = Boolean.parseBoolean(getParameter(request, "industry-professional", "false"));
     boolean recruiter = Boolean.parseBoolean(getParameter(request, "recruiter", "false"));
```

Comment:
Sorry just noticed this, but does `boolean other` actually end up getting used anywhere?

Reply:
Nope! I just deleted the line.